<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Book_Information</class>
 <widget class="QWidget" name="Book_Information">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="photo">
   <property name="geometry">
    <rect>
     <x>368</x>
     <y>30</y>
     <width>180</width>
     <height>240</height>
    </rect>
   </property>
   <property name="text">
    <string>图片</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>301</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="book_name">
      <property name="text">
       <string>书名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="writer">
      <property name="text">
       <string>作者</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="publisher">
      <property name="text">
       <string>出版社</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="publish_time">
      <property name="text">
       <string>出版时间</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="borrow_number">
      <property name="text">
       <string>借阅数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="collection_number">
      <property name="text">
       <string>收藏数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="price">
      <property name="text">
       <string>价格</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="storage_number">
      <property name="text">
       <string>库存量</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="left_number">
      <property name="text">
       <string>剩余本数</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="description">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>301</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>详细描述</string>
   </property>
  </widget>
  <widget class="QPushButton" name="apoint_book">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>预约</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mark_book">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>300</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>收藏</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
